quickbuild_whitelist

```diff
@@ -4,5 +4,6 @@ redragon/k556
 keychron/k2/rgb
 keychron/k4/rgb/v1
 keychron/k4/rgb/v2
+keychron/k7/rgb
```

Comment:
committing to testing regularly I see :stuck_out_tongue_closed_eyes:

Reply:
yeah i dont mind for now. trying to stay in the loop of latest changes.